packages/common/base/src/types/utils.ts

```diff
@@ -1,4 +1,4 @@
 export type ObjectValues<T extends object> = T[keyof T];
 export function objectValues<T extends object>(obj: T): ReadonlyArray<T[keyof T]> {
-    return Object.values(obj);
+    return Object.freeze(Object.values(obj));
```

Comment:
Freeze object over just relying on "type safety" to match the expected behaviour